<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCarTest</class>
 <widget class="QDialog" name="DialogCarTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车检表</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>车辆检测表行号</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>84</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>出入口</string>
   </property>
   <widget class="QRadioButton" name="radioout">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>30</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>出口</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioin">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>30</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>入口</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionull">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>32</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>空</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>56</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>651</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>相位</string>
   </property>
   <widget class="QCheckBox" name="bao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位1 </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>27</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>64</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_12">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位13</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_14">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_15">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位15</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_16">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>64</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位16</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butfind">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butok">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butquery">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>283</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询
已设</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>296</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>已设行数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butclearall">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>283</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空
全表</string>
   </property>
  </widget>
  <widget class="QLabel" name="hang">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butok_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
